core/templates/pages/story-editor-page/editor-tab/story-node-editor.directive.html

```diff
@@ -10,6 +10,15 @@
       </i>
     </span>
   </div>
+  <div class="story-node-field">
+    <label for="storyNodeDescription" class="form-heading">Chapter Description</label>
+    <input id="storyNodeDescription" type="text" class="form-control protractor-test-add-chapter-description"
```

Comment:
Character limit is 152. Also, add a backend validation check to make sure length is less than this. Add a constant to android_validation_constants and constants.ts after the already existing length limit constants and use the added one in constants.ts to interpolate the values here.

Reply:
Done.